### DISCUSSION TITLE
Key not held when holding button

### DISCUSSION THREAD

**Discussion by Aethyrion** [AntiMicroX/antimicrox#532] [Q&A]
When I assign a key to a button, the key is not held while I hold the button down, it is only pressed once and released immediately. I am missing something?

Using the last AMD64 release.

**Comment by pktiuk** [comment]
Do you mean executing action assigned to your button, or visualizing it in app?
  
If this is problem with visualization of buttons I guess there is a problem with your gamepad.
Does it act properly with other apps? Have you tested your gamepad with other apps like qjoypad, jstest-gtk ? Does it work properly with them?

**Comment by Aethyrion** [comment]
Thank you for your answer.

The visualization is working properly, the problem is indeed executing the action assigned to the button. For example, if the "A" key is assigned to one of my gamepad buttons, holding it will only display "A" once in any text field no matter how long I hold the button down. If I hold down the "A" key on my keyboard it will display one "A" at first, but soon after other "AAAAAAAAA" will appear. Exact same issue if I assign the "VOL+" key code, the volume is increased only one step no matter how long I hold the button down..

My gamepad (DS4) is working properly anywhere else. I have just tested with another one (8bitdo SN30), it "works" exactly the same.

I must probably be doing something wrong but can't figure what.

**Reply by mplzgta** [reply]
You can enable repeat on holding a joystick 'key' by clicking the Turbo checkbox.  In addition, the Turbo settings can be further edited (timing and delay) in the dialog shown by clicking the Advanced button when configuring a joystick key.

**Comment by paperc07** [comment]
How can I make it hold down a button, like I am trying to use an aimbot I set the L Trigger to press the C key on my keyboard, cause that's the key the aimbot uses to auto aim and it works with keyboard but with controller it needs to be held, I tried the turbo button and nothing
